src/gloss/data/primitives.clj

```diff
@@ -71,6 +71,47 @@
    :be `ByteOrder/BIG_ENDIAN
    :ne `(ByteOrder/nativeOrder)})
 
+(defn- native-byte-order []
+  (if (= (ByteOrder/nativeOrder) ByteOrder/LITTLE_ENDIAN)
+    :le
+    :be))
```

Comment:
This can be a `def`, not a `defn`, since `ByteOrder/NATIVE_ORDER` is `final`.

Reply:
Fixed